File: common/src/main/java/com/yangwz/common/net/HttpEntity.java
package com.yangwz.common.net;

/**
 * @author : yangweizheng
 * @date : 2021/2/24 14:19
 */
public class HttpEntity<T extends Object> {
    private String code;
    private T data;
    private String msg;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    @Override
    public String toString() {
        return "HttpEntity{" +
                "code='" + code + '\'' +
                ", data=" + data +
                ", msg='" + msg + '\'' +
                '}';
    }
}

File: common/src/main/java/com/yangwz/common/net/HttpEntity.kt
package com.yangwz.common.net

/**
 * @author : yangweizheng
 * @date : 2021/2/24 14:19
 */
class HttpEntity<T : Any?> {
    var code: String= TODO()
    var data: T = TODO()
        private set
    var msg: String = TODO()
    fun setData(data: T) {
        this.data = data
    }

    override fun toString(): String {
        return "HttpEntity{" +
                "code='" + code + '\'' +
                ", data=" + data +
                ", msg='" + msg + '\'' +
                '}'
    }
}